packages/accounts-base/accounts_server.js

```diff
@@ -1519,11 +1519,11 @@ export class AccountsServer extends AccountsCommon {
   }
 
   _handleError = (msg, throwError = true, errorCode = 403) => {
-    const isErrorAmbiguous = this._options.ambiguousErrorMessages ?? Meteor.isProduction;
+    const isErrorAmbiguous = this._options.ambiguousErrorMessages;
```

Comment:
The default for this needs to be `true` for security purposes.

Reply:
sure, thanks for the feedback!